src/main/java/org/spongepowered/api/text/Texts.java

```diff
@@ -375,6 +506,19 @@ public static TextBuilder builder() {
     }
 
     /**
+     * Creates a new unformatted {@link TextBuilder.Placeholder} with the
+     * specified textArgument.
+     *
+     * @param textArgument The textArgument of the placeholder
+     * @return The created placeholder builder
+     * @see Text.Placeholder
+     * @see TextBuilder.Placeholder
+     */
+    public static TextBuilder.Placeholder placeholderBuilder(TextArgument textArgument) {
```

Comment:
I am not sure what you are referring at:
That there is no
`TextBuilder.Placeholder placeholderBuilder(Text text, String key)`
and
`TextBuilder.Placeholder placeholderBuilder(Text text, TextArgument textArgument)`
?

Or are you referring to the method naming?